api/bazel/api_build_system.bzl

```
@@ -54,12 +54,15 @@ def _py_proto_mapping(dep):
 # https://github.com/bazelbuild/bazel/issues/3935 and/or
 # https://github.com/bazelbuild/bazel/issues/2626 are resolved.
 def _api_py_proto_library(name, srcs = [], deps = []):
+    mapped_deps = [_py_proto_mapping(dep) for dep in deps]
+    mapped_unique_deps = []
+    [mapped_unique_deps.append(d) for d in mapped_deps if d not in mapped_unique_deps]
```

```suggestion
     mapped_unique_deps = {k: True for k in mapped_unique_deps}.keys()
```

i think the canonical way to do sets in starlark is with `dict.keys()`

---

Done